elodie/filesystem.py

```diff
@@ -589,7 +589,7 @@ def process_file(self, _file, destination, media, **kwargs):
             #  before we made any changes.
             # Then set the utime on the destination file based on metadata.
             os.utime(_file, (stat_info_original.st_atime, stat_info_original.st_mtime))
-            self.set_utime_from_metadata(media.get_metadata(), dest_path)
+            self.set_utime_from_metadata(metadata, dest_path)
```

Comment:
Looks like this is a safe change. Calling `.get_metadata()` was part of a [PR](https://github.com/jmathai/elodie/pull/331/files) where `.set_original_name()` was added and prior to that commit `metadata` wasn't stored as a variable.